crates/nu-command/src/default_context.rs

```diff
@@ -405,6 +405,21 @@ pub fn create_default_context() -> EngineState {
             MathStddev,
             MathSum,
             MathVariance,
+            Sin,
```

Comment:
The only thing that jumped out at me was tha tall these internal names start with Math and then these new ones do not start with Math. I wasn't sure if that was by design or just an oversight?

Reply:
Yeah just a bit of rebellion to the repetitiveness :D I can make that consistent.